nodes/config/ui_page.html

```diff
@@ -277,6 +291,9 @@
     <div class="form-row">
         <label for="node-config-input-layout"><i class="w-16 fa fa-th"></i> <span data-i18n="ui-page.label.layout"></label>
         <input type="text" id="node-config-input-layout">
+        <button id="node-config-layout-edit" type="button" class="red-ui-button" style="margin-top: 4px; margin-right: 5px;" title="Edit Layout">
+            <i class="fa fa-pencil-square-o"></i>
+        </button>
         <input type="hidden" id="node-config-input-layoutType">
```

Comment:
For the alignment:
```suggestion
        <div style="width: 70%; display: inline-flex;">
            <div style="flex-grow: 1;">
                <input type="text" id="node-config-input-layout" style="width: 100%;">
                <input type="hidden" id="node-config-input-layoutType">
            </div>
            <button type="button" id="node-config-layout-edit" class="red-ui-button" style="margin-left: 10px;" title="Edit Layout">
                <i class="fa fa-pencil-square-o"></i>
            </button>
        </div>
```

Don't forget to translate the title (hardcoded here) 😉

Reply:
@GogoVega 
That looks indeed very nice that way:

![image](https://github.com/user-attachments/assets/6c272636-998c-43bb-b9f3-6a9a29265179)

Really appreciated!!

> Don't forget to translate the title (hardcoded here)

Yes indeed I had also considered to translate the title, but there were other untranslated titles.  So I was thinking to myself: enough higher priority stuff to day, and Rome wasn't build in one day neither :innocent: